api4/channel.go

```diff
@@ -1074,7 +1074,15 @@ func deleteChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = c.App.DeleteChannel(channel, c.App.Session().UserId)
+	if c.Params.Permanent {
```

Comment:
Is there a PR to document this flag in the API docs?

Reply:
Yes, will open a few documentation PRs soon

Reply:
https://github.com/mattermost/mattermost-api-reference/pull/563